source/patterns/@aws-solutions-constructs/aws-iot-s3/test/integ.iot-s3-defaultprops.ts

```diff
@@ -27,8 +27,12 @@ const props: IotToS3Props = {
       sql: "SELECT * FROM 'solutions/constructs'",
       actions: []
     }
+  },
+  logS3AccessLogs: false,
```

Comment:
Just getting started on the review - at least one integ test has this turned on, right?

Reply:
yes, we have a test case where this is turned on with access bucket destroyed during stack teardown